learning/dashboard/group9/readme.md

```diff
@@ -0,0 +1,105 @@
+# How to install and setup NodeJS and npm?
+- **step 1:**
+Open the Terminal on your Ubuntu system. You can do this by pressing the **"Ctrl+Alt+T"** keys simultaneously.
+- **step 2:**
+Update your package list by typing the following command in the Terminal:
+
+   `sudo apt-get update`
+ - **step3:**
+  Install Nodejs using the following command: 
+  
+    `sudo apt get install node js`
+  - **step4:**
+   Check if Node.js is installed by typing the following command:
+   
+    `node -v`
+    
+    This will display the version number of Node.js installed on your system.
+   - **step5:**
+Install NPM (Node Package Manager) by typing the following command:
+
+      `sudo apt-get install npm`
+   - **step6:**
+    Check if NPM is installed by typing the following command:
+
+      `npm -v`
+   - **step7:**
+      If you encounter any issues with the installation, you can try updating the package list again by typing the following command:
+      
+        `sudo apt-get update`
+
+       Then repeat steps 3-6.
+## Steps to update the nodejs and npm versions
+
+- **step 1:**
+Open the Terminal on your Ubuntu system.
+- **step 2:**
+Check the currently installed versions of Node.js and NPM by typing the following commands:
+
+   `node -v npm -v`
+ - **step3:**
+ To update Node.js, you can use the Node Version Manager (nvm) tool. If you do not have nvm installed, you can install it using the following command:
```

Comment:
this section should contain steps to update NodeJS and NPM without using NVM.

Reply:
in the question, it was mentioned with using nvm, we will surely add without using nvm too

Reply:
- Install appropriate version of NodeJS and Npm based on the OS and version
    - How to update the NodeJS and npm versions **-> This is the current section. Thisis without NVM**
    - How to manage multiple versions of NodeJS using [nvm](https://github.com/nvm-sh/nvm) **-> This is the next section.**